content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
   default:
     identifier: automations
     parent: w-b-models
-title: Automations
+title: Create and manage automations
 weight: 4
 ---
 
+This page describes _automations_ in W&B and shows how to create and manage them. Create an automation to trigger workflow steps, such as automated model testing and deployment, based on an event in W&B, such as when an [artifact]({{< relref "/guides/core/artifacts" >}}) or a [registered model is changed]({{< relref "/guides/models/registry/" >}}), or based on the results of a [run]({{< relref "/guides/models/track/runs/manage-runs.md" >}})
+
+An automation defines the [event scopes and types]({{< relref "#event-scopes-and-types" >}}) to watch for and the [action]({{< relref "#event-actions" >}}) to take when the event occurs, such as running a webhook or posting to a Slack channel.
+
+## Event scopes and types
+
+An automation's triggering event depends on the automation's scope.
+
+### Registry
+For a [registered model]({{< relref "/guides/models/registry/">}}), you can configure an automation to run on these events:
+
+- **Linking a new artifact to a registered model**: Test and validate new model candidates.
+- **Adding a new alias to a version of the registered model**: Trigger a special step of your workflow when a new model version has a label or alias applied. For example, deploy a model when it has the `deploy` alias applied.
+
+### Project
+For a model in a project that is not in a registry, you can configure an automation to run on these events:
+
+- **A new version of an artifact is created in a collection**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created.
+- **An artifact alias is added**: Apply recurring actions when a specific alias is applied to an artifact version. For example, run a series of downstream processing steps when an artifact gains the `test-set-quality-check` alias.
+
+### Run
+You can configure an automation to run based on the results of a run:
+
+- **Run metric threshold met**: Apply recurring actions based on a run metric's value and whether it is above, below, or exactly equal to the threshold. Optionally, specify how to average results.
+- **Run metric change threshold met**: Apply recurring actions based on whether a run metric's amount of change (absolute or relative) is above, below, or exactly equal to the threshold. Optionally, specify how to average results.
```

Comment:
the options for change thresholds are different enough that the descrption should be different - eg the amount of change is no above/below/exactly equal to

Reply:
I'll implement this in the followup PR.